src/bp/admin/ui/src/app/BottomPanel/Logs/index.tsx

```diff
@@ -183,7 +183,8 @@ class BottomPanel extends React.Component<Props, State> {
 
   render() {
     const allLogs = [...this.state.initialLogs, ...this.logs]
-    const filtered = this.state.botFilter === '*' ? allLogs : allLogs.filter(x => x.botId === this.state.botFilter)
+    const filtered =
+      this.state.botFilter === '*' ? allLogs : allLogs.filter(x => x.botId === this.state.botFilter || !x.botId)
```

Comment:
I personally find it kind of hard to read!

Can you maybe move this piece of code (`allLogs.filter(x => x.botId === this.state.botFilter || !x.botId)`) into an function named something like `filterByBotId`? Then maybe add a comment mentioning that it keeps global (`!x.botId`) and logs matching a certain filter (`x.botId === this.state.botFilter`)

Reply:
Good point, moved it to a variable